src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -394,6 +394,10 @@ private void ShowHelp()
             Dbg.Assert(_helpText != null, "_helpText should not be null");
             _hostUI.WriteLine("");
             _hostUI.Write(_helpText);
+            if (_showExtendedHelp)
+            {
+                _hostUI.Write(ManagedEntranceStrings.ExtendedHelp);
```

Comment:
Shouldn't this show both `UsageHelp` and `ExtendedHelp`?

Reply:
Line 396 already show `UsageHelp`.

Reply:
Got it!